src/org/usfirst/frc/team2335/robot/commands/AutoFromStation1.java

```diff
@@ -0,0 +1,16 @@
+package org.usfirst.frc.team2335.robot.commands;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class AutoFromStation1 extends CommandGroup
+{ 
+    public AutoFromStation1() 
+    {
+    	//TODO: get correct time and distance values for the following. Current values are placeholders.
```

Comment:
Hey, I made that comment! Do we still need this or are our values no longer placeholders?